.gitignore

```diff
@@ -3,3 +3,4 @@
 node_modules
 coverage
 dist
+.idea
```

Comment:
Can you add this in a separate PR instead? I'm sorry if being too picky here, but I just want to have the PR's a bit organized :)

Reply:
Yes, no worries. It's a good idea ;)